### ISSUE TITLE
Problem with mu4e frame

### ISSUE THREAD

**Issue by prosoitos** [ch11ng/exwm#431]
*Non-urgent as I found a workaround*.

When running mu4e in exwm, if I choose the option to compose messages **not** in a new frame, the message window is totally off in the low-right corner, with mode-line, minibuffer, and part of the window out of the screen. Upon sending the message, the window remains the same:

![image](https://user-images.githubusercontent.com/4634851/40391347-c4d3196c-5dcc-11e8-9596-120361270bb3.png)

and the only solutions out of that that I have found are to reboot or to start working in a new workspace.

If I choose the mu4e option to compose messages in a new frame, that frame is off to the side, but entirely visible. Upon sending the message, I end up with the same window in the low-right corner, but re-selectioning the current workspace resets the window and brings me back to normal. So not really pretty, but with this option, it works.

**Comment by ch11ng** [comment]
You definitely should not set `mu4e-compose-in-new-frame` or you'll end up with a new workspace each time you compose a new mail.  As I've just tried mu4e just works fine.  How do you set up mu4e?

**Comment by prosoitos** [comment]
Ah! This is totally my bad: while using openbox, I had set a default size and position for `mu4e-compose` frames (and of course totally forgot about it). Your message pushed me to dig in all my mu4e settings and I found this long forgotten line of code in my init file. I have now switched `mu4e-compose-in-new-frame` to `nil` and all is working great.

Apologies for this and thank you for your comment which sent me in the right direction (which I should have taken by myself before posting an issue...).

I guess I will close this issue then.

Note: thank you also for the info about new frames in mu4e creating new workspaces. I was a little puzzled about what was going on with my workspaces and I hadn't clued into where the cause was. This explains it too.

Thank you very much for your time!!!
